build-tools/automation/build.groovy

```diff
@@ -137,6 +216,10 @@ timestamps {
             sh "make -C ${XADir} -k package-test-errors"
 
             def publishTestFilePaths = "${XADir}/xa-test-errors*"
+            if (isPr) {
+                publishTestFilePaths = "${publishTestFilePaths},${XADir}/test-errors.zip"
```

Comment:
We should remove this divergence between PR & master builds, and always generate & upload `test-errors.zip`, even in master builds.

...though where is `test-errors.zip` coming from?  I think this might be a "legacy" value that isn't being generated anymore.

Reply:
We use `test-errors.zip` it has files from failing MSBuild tests.

`make package-test-errors` zips up bin/TestDebug/temp and a few others.

Reply:
Divergence removed with commit https://github.com/xamarin/xamarin-android/pull/2767/commits/fe6cef595b77038b306bca0cc7551f0a7cc0846a